lib/registration.pm

```diff
@@ -408,7 +408,7 @@ sub fill_in_registration_data {
                     wait_screen_change { send_key 'alt-a' };
                     while (
                         assert_screen(
-                            ['yast_scc-license-dialog', 'yast_scc-automatic-changes', 'yast_scc-prompt-reboot', 'yast_scc-installation-summary'], 900
+                            ['yast_scc-license-dialog', 'yast_scc-automatic-changes', 'yast_scc-prompt-reboot', 'yast_scc-installation-summary'], 1800
```

Comment:
Better to add a comment to describe that why you en-large the timeout

Reply:
Done.